tests/lowering/misc/test_select.py

```diff
@@ -0,0 +1,40 @@
+import torch
+import torch_ttnn
+import pytest
+import ttnn
+
+from tests.utils import assert_with_pcc
+
+
+class SelectModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, input):
+        return torch.select(input, 3, 0)
+
+
+@pytest.mark.parametrize(
+    "input_shapes",
+    [(1, 32, 16, 3, 96)],
+)
+def test_adaptive_avg_pool_2d(device, input_shapes):
```

Comment:
Rename to `test_select`?

Reply:
done. refer to 5043623e7007c31b066987695337e96cd3878473